pages/guides/docker_containerized_builds.md.erb

```diff
@@ -16,9 +16,9 @@ app:
   volumes:
     - ./:/app
   links:
-    - db:db
-    - redis:redis
-    - memcache:memcache
+    - db
```

Comment:
Nice, back in the day you'd end up with only `redis_1` in `/etc/hosts` but I just checked and they now include a non-numbered one for the first service with that name:

```bash
$ cat docker-compose.yml 
app:
  image: ubuntu
  links:
    - redis
redis:
  image: redis
$ docker-compose run app bash -c 'grep redis /etc/hosts'
172.17.0.1	redis_1 8cb1d3e600ae composetest_redis_1
172.17.0.1	composetest_redis_1 8cb1d3e600ae
172.17.0.1	redis 8cb1d3e600ae composetest_redis_1
172.17.0.1	composetest_redis_1
172.17.0.1	composetest_redis_1.bridge
```

:+1: no need for `redis:redis` anymore